packages/pillarbox-debug-panel/src/metric-label.js

```diff
@@ -0,0 +1,59 @@
+import videojs from 'video.js';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/component').default}
+ */
+const Component = videojs.getComponent('Component');
+
+/**
+ * Represents a MetricLabel component for displaying individual metrics.
+ *
+ * The MetricLabel is a simple component used to display a text label for a specific metric.
+ * It shows the metric's name and its current value.
+ */
+class MetricLabel extends Component {
+  /**
+   * Creates an instance of a MetricLabel.
+   *
+   * @param {import('video.js/dist/types/player.js').default} player - The Video.js player instance.
+   * @param {Object} options - Configuration options for the MetricLabel.
+   * @param {string} options.label - The label text for the metric. This text is displayed along with the metric value.
+   */
+  constructor(player, options) {
+    super(player, options);
+  }
+
+  /**
+   * Constructs the DOM element that will represent the MetricLabel in the player UI.
+   *
+   * @param {string} [tag='div'] - The HTML tag name for the element.
+   * @param {Object} [props={}] - Additional properties to set on the element.
+   * @param {Object} [attributes={}] - Additional attributes to set on the element.
+   *
+   * @return {Element} The created DOM element.
+   */
+  createEl(tag = 'div', props = {}, attributes = {}) {
+    props = Object.assign({ className: this.buildCSSClass() }, props);
+
+    return videojs.dom.createEl(tag, props, attributes);
+  }
+
+  /**
+   * Updates the value of the metric displayed by the MetricLabel.
+   *
+   * @param {string|number} value - The current value of the metric to display.
+   */
+  update(value) {
+    videojs.dom.textContent(this.el_, `${this.options_.label}: ${value}`);
```

Comment:
No `this.localize` ?

Reply:
The debug panel is a technical component intended for development use. I don't see the benefit of localising this, I can add "this.localize" but adding the actual translations seems a bit too much.